src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* New instances of `OpenTelemetryLoggerOptions` will no longer be created during
```

Comment:
I guess the description (which is phrased as a behavior change) here might be confusing to users - most of them probably don't even know about `OpenTelemetryLoggerOptions`.

Consider being more explicit here by starting with "Fixed an issue ..." (whether `OpenTelemetryLoggerOptions` will be created or not is an implementation detail that we might change later if we figured out how to properly support dynamic configuration reload).

Reply:
New version pushed